src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -316,6 +316,46 @@ struct P1_ALIGNAS(4) RawIMUOutput : public MessagePayload {
   int32_t gyro[3] = {INT32_MAX, INT32_MAX, INT32_MAX};
 };
 
+/**
+ * @brief IMU sensor measurement input (@ref MessageType::IMU_INPUT,
+ *        version 1.0).
+ * @ingroup measurement_messages
+ *
+ * This message is an input from the device containing raw IMU acceleration and
+ * rotation rate measurements.
+ *
+ * See also @ref IMUInput.
+ */
+struct P1_ALIGNAS(4) IMUInput : public MessagePayload {
```

Comment:
For consistency, we've been ordering the classes input, output, raw output. Please move this class up above `IMUOutput` to match the other measurement types.